docs/pages/User/GettingStarted/test.md

```diff
@@ -22,75 +21,110 @@ kubectl create namespace liqo-demo
 
-In case the `nodeSelector` is not specified, the Kubernetes scheduler will select the best hosting node based on the available resources, which can be either a node in the *home* cluster or in the *foreign* cluster.
+In case the affinity constraint is not specified, the Kubernetes scheduler selects the best hosting node based on the available resources.
+Hence, each pod can be scheduled either in the *home* cluster or in the *foreign* cluster.
 
 {{% /expand%}}
 
-Now you can check the state of your pod; the output should be similar to the one below, confirming that the pod is running on a virtual node (i.e. a node named `liqo-<...>`):
+Now you can check the state of your pods.
+The output should be similar to the one below, confirming that one `nginx` pod is running locally, while the other on a virtual node (i.e. a node named `liqo-<...>`):
 
 ```
 kubectl get pod -o wide -n liqo-demo
-NAME    READY   STATUS    RESTARTS   AGE   IP           NODE                                      NOMINATED NODE   READINESS GATES
-nginx   1/1     Running   0          41m   10.45.0.12   liqo-1dfa22f9-1cdd-4401-9e7a-c5342ec90059   <none>           <none>
+
+NAME           READY   STATUS    RESTARTS   AGE   IP              NODE
+nginx-local    1/1     Running   0          76s   10.244.2.214    worker-node-1
+nginx-remote   1/1     Running   0          76s   172.16.97.219   liqo-9a596a4b-591c-4ac6-8fd6-80258b4b3bf9
 ```
 
 ## Check the pod connectivity
 
-First, let's retrieve the IP address of the `nginx` pod:
+First, let's retrieve the IP address of the `nginx` pods:
 
 ```bash
-POD_HOST=$(kubectl get pod nginx -n liqo-demo --template={{.status.podIP}})
-echo $POD_HOST
+LOCAL_POD_IP=$(kubectl get pod nginx-local -n liqo-demo --template={{.status.podIP}})
+REMOTE_POD_IP=$(kubectl get pod nginx-remote -n liqo-demo --template={{.status.podIP}})
+echo "Local Pod IP: ${LOCAL_POD_IP} - Remote Pod IP: ${REMOTE_POD_IP}"
 ```
 
-If you have direct connectivity with the cluster (e.g. K3s) from your host, you can open a browser and directly connect to the value of `$POD_HOST`; alternatively you can also use the following `curl` command:
-
+If you have direct connectivity with the home cluster from your host (e.g. you are running K3s locally),
+you can open a browser and directly check the connectivity to both IP addresses.
+Similarly, you can also use `curl`:
 ```
-curl --verbose $POD_HOST
+curl ${LOCAL_POD_IP}
+curl ${REMOTE_POD_IP}
 ```
 
 If you do not have direct connectivity, on the other hand, you can fire up a pod and run `curl` from inside:
-
 ```
-kubectl run --image=curlimages/curl tester -n default -it --rm -- curl --verbose $POD_HOST
+kubectl run --image=curlimages/curl curl -n default -it --rm --restart=Never -- curl ${LOCAL_POD_IP}
+kubectl run --image=curlimages/curl curl -n default -it --rm --restart=Never -- curl ${REMOTE_POD_IP}
 ```
```

Comment:
We're missing a conclusion here: what do we expect the user to see? That both output are the same? That you can reach both? Anything else?